packages/server/src/routes.ts

```diff
@@ -18,15 +18,20 @@ export const createRouter = (ctx: AppContext): express.Router => {
     res.send({ version })
   })
 
-  // @TODO paginate & test this
+  // Export ops in the form of paginated json lines
   router.get('/export', async function (req, res) {
-    const fullExport = await ctx.db.fullExport()
+    const parsedCount = parseInt(req.query.count)
+    const count = isNaN(parsedCount) ? 1000 : Math.min(parsedCount, 1000)
```

Comment:
To help avoid accepting weird stuff including numbers written in hex for the `count`:
```suggestion
    const { count } = req.query
    const parsedCount = count ? parseInt(count, 10) : 1000
    if (isNaN(parsedCount) || parsedCount < 1) {
      throw new ServerError(400, 'Invalid count parameter')
    }
    const count = Math.min(parsedCount, 1000)
```

Reply:
nice i like it 👍